ddp/ddp_pipeline/src/main/java/org/mskcc/cmo/ks/ddp/pipeline/DDPSeqDateTasklet.java

```diff
@@ -123,6 +130,7 @@ public Map<String, Date> getFirstSeqDatePerPatientFromFile(String filename, Buff
                 LOG.warn(warnings.get(w)); 
             }
         }
-        return patientFirstSeqDateMap;
+        DDPUtils.setSampleSeqDateMap(sampleSeqDateMap);
+        DDPUtils.setPatientFirstSeqDateMap(patientFirstSeqDateMap);
```

Comment:
we are setting a class data members in a remote class in these two function calls. This is equivalent to using a global variable to pass data around in a program - somewhat error prone but not invalid. Also, we use DDPUtils to hold several other of these global mappings ... so that they can later be used for formatting records I believe, so this is in the current pattern of usage. So ok.